### PR TITLE
CSS-Three-Column-Layout challenge - Alvaro Diaz

### PR DESCRIPTION
**Pull request by alvarodiazjr**
Screen Shot 2022-06-30 at 8 18 02 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Alvaro! The text in each of your columns is too wide as compared to the example, please revise and submit a new screenshot with your revision

**Comment by alvarodiazjr** [comment]
Screen Shot 2022-07-01 at 1 25 48 PM

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  flex-basis: calc(100%/3);
+}
+
+.column-two-thirds{
+  flex-basis: calc(100%/3);
+}
+
+.column-third{
+  flex-basis: calc(100%/3);
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css]: Why do you need a `column-one-third` and ` column-third` class? Also, the measurement for `column-two-thirds` is not accurate. These details **_really_** matter.

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+.justify-center{
+  padding: 5px;
+  justify-content: center;
+}
+
+.justify-center h3, p{
+  padding: 5px;
+  justify-content: center;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css]: You should check if the `justify-content` declaration here is necessary. If the property does nothing, it should be removed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-three-column-layout/styles.css [hunk 3]
```diff
@@ -12,15 +12,7 @@
...
-.column-one-third{
-  flex-basis: calc(100%/3);
-}
-
-.column-two-thirds{
-  flex-basis: calc(100%/3);
-}
-
-.column-third{
+.column{
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css]: No, sorry, this is not what I intended. The name `column` give me no indication of its purpose. You need a column class for every size of `column` you use in your layout and a name that specifies its size. You should review the `css-layout-classes` exercise in depth to get a refresher on the concept.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**